app/serializers/listing_index_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class ListingIndexSerializer < ActiveModel::Serializer
+  include Rails.application.routes.url_helpers
+  attributes :id, :category, :description, :scene, :height, :address, :price
```

Comment:
title of listing or equivalent required for 'pitching' your listing

Reply:
we changed that to just Id, category and lead plus the Scene that shows if the car park is inside or outside.